conduct.md

```diff
@@ -4,66 +4,85 @@ layout: page
 
 ## Code of Conduct
 
-DjangoCon Europe aims to be a welcoming event where people meet in a friendly environment.
-Accordingly, all participants are expected to show respect and courtesy to other participants
-throughout the conference.
+All delegates/attendees, speakers, exhibitors, organizers and
+volunteers at DjangoCon Europe 2018 are required to conform to the following Code of Conduct. Organizers will enforce this code before and throughout the event (including related activities such as the social events).
 
-To make clear what is expected, all delegates/attendees, speakers, exhibitors, organizers and
-volunteers at DjangoCon Europe 2018 event are required to conform to the following Code of Conduct.
-Organizers will enforce this code before and throughout the event (including related activites such
-as the social events).
+## Why do we have a Code of Conduct?
 
-### In short
+Our goals with having this CoC are:
```

Comment:
I'd prefer calling it "Code of Conduct" throughout, as CoC is an abbreviation not commonly known to all.

Reply:
I've introduced the abbreviation explicitly in the first paragraph. In other uses, I've gone for CoC only where it would otherwise be very repetitive.